src/app/scoreboard/scoreboard.js

```diff
@@ -17,7 +19,12 @@ class Scoreboard extends React.PureComponent {
             <Table>
                 <TableHead>
                     <TableRow>
-                        {['Name', 'Score', 'AC'].concat(this.props.problems)
+                        {[
+                            <LocalizedMessage id="scoreboard.table.head.name" />,
+                            <LocalizedMessage id="scoreboard.table.head.score" />,
+                            <LocalizedMessage id="scoreboard.table.head.accepted-count" />
+                        ]
+                            .concat(this.props.problems)
                             .map((entries, index) => <TableCell key={`head_${index}`}>{entries}</TableCell>)}
```

Comment:
This piece did good, but need some refactors

Reply:
Maybe I did it with ae83941b325221c83d5aa7ac9f8e25510518e82f? If you say about #127, then let this PR be itself. Maybe until I can confirm that it will not break anything.